packages/dashboard-frontend/src/store/SanityCheck/index.ts

```diff
@@ -107,7 +107,7 @@ export const actionCreators: ActionCreators = {
             delay(1000);
           }
         }
-      } catch (e) {
+      } catch (e: any) {
```

Comment:
```suggestion
      } catch (e: unknown) {
```
TypeScript won't let us perform arbitrary operations on values of type unknown.

WDYT?

Reply:
This wont work with the `e.response.data.trace.join('\n')`

Reply:
you need to use the type guard to guarantee the error object type:

```typescript
        if (
          helpers.errors.includesAxiosResponse(e) &&
          e.response.data.trace &&
          Array.isArray(e.response.data.trace)
        ) {
          console.error(e.response.data.trace.join('\n'));
        }
```

Reply:
thanks @olexii4, done